packages/context/README.md

```diff
@@ -97,17 +97,42 @@ const configuration = {
 export default configuration;
 ```
 
-### How to consume : react api context consumer method (Layout/Header.js)
+### How to consume : Hooks method (Pages/Dashboard/Dashboard.js)
 
-"AuthenticationConsumer" component inject to children "props" the properties :
+"useReactOidc" returns all props from the Hook :
+
+```javascript
+import React from 'react';
+import { useReactOidc } from '@axa-fr/react-oidc-context';
+
+const Dashboard = () => {
+  const { oidcUser } = useReactOidc();
+  const { profile } = oidcUser;
+  return (
+    <div>
+      <h1>Dashboard</h1>
+      <p>Protected Dashboard</p>
+      <span>
+        Hello {profile.given_name} {profile.family_name}
+      </span>
+    </div>
+  );
+};
+
+export default Dashboard;
+
+```
+### How to consume : react api context method (Layout/Header.js)
+
+"AuthenticationContext" context contains all props you need
 
 - oidcUser : user information (null if not authenticated)
 - logout: logout function
 - login: login function
 
 ```javascript
 import React from "react";
-import { AuthenticationConsumer } from "@axa-fr/react-oidc-context";
+import { AuthenticationContext } from "@axa-fr/react-oidc-context";
```

Comment:
so we will deliver a first major update? :p

Reply:
Yes Major update !